src/utils/notificationsLocalStorage.ts

```diff
@@ -55,41 +95,84 @@ export const setProcessError = (error: unknown) => {
 };
 
 export const setProcessEnded = () => {
+  const accountIdLS = getLSAccountId();
+  const localStorageByAccountId = getNotificationLocalStorage();
+
   localStorage.setItem(
     NOTIFICATIONS_STORAGE,
     JSON.stringify({
-      ...getNotificationLocalStorage(),
+      ...getNotificationLocalStorageFull(),
+      [accountIdLS]: {
+        ...localStorageByAccountId,
+        subscribeStarted: false,
+      },
       subscribeStarted: false,
     }),
   );
 };
 
 export const setProcessStarted = () => {
+  const accountIdLS = getLSAccountId();
+  const localStorageByAccountId = getNotificationLocalStorage();
+
   localStorage.setItem(
     NOTIFICATIONS_STORAGE,
     JSON.stringify({
-      ...getNotificationLocalStorage(),
+      ...getNotificationLocalStorageFull(),
+      [accountIdLS]: {
+        ...localStorageByAccountId,
+        permission: false,
+        subscribeStarted: true,
+      },
       permission: false,
       subscribeStarted: true,
     }),
   );
 };
 
 export const setClearData = () => {
-  localStorage.setItem(NOTIFICATIONS_STORAGE, JSON.stringify({}));
+  const accountIdLS = getLSAccountId();
+
+  localStorage.setItem(
+    NOTIFICATIONS_STORAGE,
+    JSON.stringify({
+      ...getNotificationLocalStorageFull(),
+      [accountIdLS]: {},
+    }),
+  );
 };
 
 export const setNotificationsSessionStorage = () => {
+  const accountIdLS = getLSAccountId();
+  const localStorageByAccountId = getNotificationLocalStorage();
+
   localStorage.setItem(
     NOTIFICATIONS_STORAGE,
     JSON.stringify({
-      ...getNotificationLocalStorage(),
+      ...getNotificationLocalStorageFull(),
```

Comment:
@marcinbodnar are these local storage changes related to supporting iOS notifications or do they serve another need?

Reply:
These are changes from different branch, it should be refreshed in few minutes.